homework/object-oriented-programming/src/main/java/com/cbfacademy/shapes/Shapes.java

```diff
@@ -0,0 +1,31 @@
+package com.cbfacademy.shapes;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Shapes {
+
+    public void computeArea(List<Enclosure> enclosure){
+       //ArrayList<Double> shapesArea = new ArrayList<>();
+        //take a list of object that implement enclosure - as a parameter
+        // use area method on this group of objects then add the area for each shape together   
+        for (int i = 0; i < enclosure.size(); i++) {
+           //
+        }
+
+        }
+        
+
+    public void computePerimeter(List<Enclosure> enclosure){
+        for (int i = 0; i < enclosure.size(); i++){
+//
+        }
+        // //take a list of object that implement enclosure 
+        // use perimeter method on this group of objects     
+    }
```

Comment:
Once you've implemented the `computeArea` method, you can just replicate the logic to compute the total perimeter